app/src/main/java/com/android/solvit/provider/ui/request/ListRequestsFeed.kt

```diff
@@ -0,0 +1,434 @@
+
+  val backgroundColor = if (selectedText != "Service") Color(0xFFFFFAF5) else Color(0xFFFFFFFF)
+  val borderTextColor = if (selectedText != "Service") Color(0xFFFFCD55) else Color(0xFF00C853)
+
+  Box(
+      modifier =
+          Modifier.testTag("ServiceChip").chipModifier(backgroundColor, borderTextColor).clickable {
+            showDropdown = !showDropdown
+          },
+      contentAlignment = Alignment.Center) {
+        Text(
+            text = selectedText,
+            fontSize = 16.sp,
+            lineHeight = 34.sp,
+            fontWeight = FontWeight(400),
+            color = borderTextColor)
+      }
+
+  DropdownMenu(
+      expanded = showDropdown,
+      onDismissRequest = { showDropdown = false },
+      modifier =
+          Modifier.border(1.dp, Color.Gray, RoundedCornerShape(12.dp))
+              .background(Color.White, RoundedCornerShape(12.dp))
+              .testTag("ServiceDropdown")) {
+        Services.entries.forEach { service ->
+          val serviceName =
+              service.name.replace("_", " ").lowercase().replaceFirstChar { it.uppercase() }
+          DropdownMenuItem(
+              modifier = Modifier.testTag(serviceName),
+              text = { Text(serviceName) },
+              onClick = {
+                selectedText = serviceName
+                onServiceSelected(serviceName)
+                showDropdown = false
+              })
+        }
+      }
+}
+
+@Composable
+fun FilterChip(label: String, isSelected: Boolean, onSelected: (Boolean) -> Unit) {
+  val backgroundColor = if (isSelected) Color(0xFFFFFAF5) else Color(0xFFFFFFFF)
+  val borderTextColor = if (isSelected) Color(0xFFFFCD55) else Color(0xFF00C853)
+
+  val context = LocalContext.current
+  Box(
+      modifier =
+          Modifier.chipModifier(backgroundColor, borderTextColor).clickable {
+            onSelected(!isSelected)
+            Toast.makeText(context, "Not Yet Implemented", Toast.LENGTH_LONG).show()
+          },
+      contentAlignment = Alignment.Center) {
+        Text(
+            text = label,
+            fontSize = 16.sp,
+            lineHeight = 34.sp,
+            fontWeight = FontWeight(400),
+            color = borderTextColor)
+      }
+}
+
+// Extension function for common modifier styling
+private fun Modifier.chipModifier(backgroundColor: Color, borderTextColor: Color) =
+    this.padding(8.dp)
+        .border(3.dp, borderTextColor, shape = RoundedCornerShape(12.dp))
+        .background(backgroundColor, shape = RoundedCornerShape(12.dp))
+        .padding(12.dp, 6.dp)
+
+@Composable
+fun ListRequestsFeedScreen(
+    serviceRequestViewModel: ServiceRequestViewModel =
+        viewModel(factory = ServiceRequestViewModel.Factory),
+    navigationActions: NavigationActions
+) {
+  val requests by serviceRequestViewModel.requests.collectAsState()
+  val selectedFilters = remember { mutableStateOf(setOf<String>()) }
+  var selectedService by remember { mutableStateOf("Service") }
```

Comment:
Nit: This just seems a little weird to me since we're not monitoring the string but rather the type of service selected. 
 
```suggestion
  var selectedService by remember { mutableStateOf("Service") }
  var selectedService by remember { mutableStateof(
```

Reply:
It is a deliberate choice to enable the addition of different types of filters in the future (by proximity or deadline for example).